Source/WebCore/dom/Element.cpp

```diff
@@ -768,8 +769,13 @@ Vector<String> Element::getAttributeNames() const
 
 bool Element::hasFocusableStyle() const
 {
-    if (renderer() && renderer()->isSkippedContent())
-        return false;
+    if (renderer() && renderer()->isSkippedContent()) {
```

Comment:
This block should be wrapped up in a function so it's easier to tell what it's doing.

Reply:
I did not put much effort in this code since I think we should go with https://github.com/WebKit/WebKit/pull/15682, which simplifies Element::hasFocusableStyle.